index.html

```diff
@@ -0,0 +1,29 @@
+<head>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">The Amazing Leo Messi!!!</h1>
+        <p id="p1">The man who became the best soccer player worldwide</p>
```

Comment:
I think it would be better to align the picture with the title and subtitle 📳.
Kindly choose the proper tag for the subtitle. <p> tags are used to add paragraphs. You´d better use <h2> tag 👍🏻

Reply:
Great job @Juanpa8830 👍 for alighting the issues and pointing out the better approach 💯, however,  you also need to state the reason (WHY) the changes are necessary so that the code author will know why making those changes is necessary. 😎